Add X.509 guidance (#21)
Azure/azure-sdk-for-c-arduino: examples/Azure_IoT_Central_ESP32/readme.md

    ![IoT Central create a device](media/iotcentraldevice-connection-info.png)

    The information provided above will create a device with **Symmetric Key** authentication. If you prefer, you can also use **X.509 certificates**. In order to do that:
     - In the `Authentication type` box, select `individual enrollment`

>      - In the `Authentication method` box, select `Certificates (X.509)`

Reviewer: ```suggestion
     - In the `Authentication type` box, select `individual enrollment`.
     - In the `Authentication method` box, select `Certificates (X.509)`.
```

     - Click on the blue folder icon for both primary and secondary and select your device Certificate.

>         > If needed, instructions on how to create an X.509 cert for tests can be found [here](https://github.com/Azure/azure-sdk-for-c/blob/main/sdk/samples/iot/docs/how_to_iot_hub_samples_linux.md#configure-and-run-the-samples) (Step 1). Please note that you might need to install some of the [prerequisites](https://github.com/Azure/azure-sdk-for-c/blob/main/sdk/samples/iot/docs/how_to_iot_hub_samples_linux.md#prerequisites) like OpenSSL.

Reviewer: ```suggestion
        > If needed, instructions on how to create an X.509 cert for testing can be found [here](https://github.com/Azure/azure-sdk-for-c/blob/main/sdk/samples/iot/docs/how_to_iot_hub_samples_linux.md#configure-and-run-the-samples) (Step 1). Please note that you might need to install some of the [prerequisites](https://github.com/Azure/azure-sdk-for-c/blob/main/sdk/samples/iot/docs/how_to_iot_hub_samples_linux.md#prerequisites) like OpenSSL.
```

4. Configure the sample.

    Enter your Azure IoT Hub and device information into the sample's `iot_configs.h`:
    - Add your Wi-Fi SSID to `IOT_CONFIG_WIFI_SSID`
    - Add your Wi-Fi password to `IOT_CONFIG_WIFI_PASSWORD`

>     - Add you IoTHub Name to `DPS_ID_SCOPE`

Reviewer: ```suggestion
    - Add you Device Provisioning Scope ID to `DPS_ID_SCOPE`
```

    - Add your Device ID to `IOT_CONFIG_DEVICE_ID`
    - If using **X.509 Cert**:
        - Uncomment the `#define IOT_CONFIG_USE_X509_CERT`
        - Add your cert to `IOT_CONFIG_USE_X509_CERT`

>         - Add your cert PK to `IOT_CONFIG_DEVICE_CERT_PRIVATE_KEY`

Reviewer: ```suggestion
        - Add your cert private key to `IOT_CONFIG_DEVICE_CERT_PRIVATE_KEY`
```
